_sass/foundation-components/_type.scss

```diff
@@ -472,12 +472,17 @@ $align-class-breakpoints:
       .print-only { display: none !important; }
       @media print {
         * {
-          background: transparent !important;
```

Comment:
@gvwilson This is why the logo doesn't show up. The logo is white and this makes the background also white.